rgf/rgf.py

```diff
@@ -3,12 +3,21 @@
 from glob import glob
 import numpy as np
 from sklearn.base import BaseEstimator, ClassifierMixin, RegressorMixin
-
+import platform
 ## Edit this ##################################################
 
+sys_name = platform.system()
+
 #Location of the RGF executable
-loc_exec = 'C:\\Users\\rf\\Documents\\python\\rgf1.2\\bin\\rgf.exe'
-loc_temp = 'temp/'
+WINDOWS = 'Windows'
+LINUX = 'Linux'
+
+if sys_name == WINDOWS:
+    loc_exec = 'C:\\Users\\rf\\Documents\\python\\rgf1.2\\bin\\rgf.exe'
+    loc_temp = 'temp/'
+elif sys_name == LINUX:
+    loc_exec = '/opt/rgf1.2/bin/rgf'
+    loc_temp = '/tmp'
```

Comment:
I got it! I was not aware of IPython problem.
Using '/tmp/rgf/' looks like nice for me.

I worried about `predict` before `fit`,
because there is a risk of using the learning results of the previous completely unrelated data set.
But it should be avoid another approach the same as sklearn models. #5